src/cpp/src/speculative_decoding/speculative_decoding_metrics.hpp

```diff
@@ -3,6 +3,8 @@
 
 #pragma once
 
+#include <iostream>
```

Comment:
it's not used in current header. If it's required in .cpp, please, move it there